Vejleder rettelser (#14)
chhoumann/p5: sections/Chapter 5 Sprint 3/planning.tex

\section{Sprint Planning}
As noted in the preceding sprint, the codebase inherited from the previous iteration was deemed problematic, costly to maintain and did not have the required components in order for the pipeline to function. 

> Therefore our sprint goal for this iteration was based on rewriting codebase to comply with the requirements and adding the endpoints in order to get the pipeline working. 

Reviewer: ```suggestion
Therefore our sprint goal for this iteration was to rewrite the codebase to comply with the requirements, and add the endpoints in order to get the pipeline working. 
```

> Based on this observation as well as the aforementioned requirements, The MVP for our part of the sprint goal was as follows:

Reviewer: ```suggestion
Based on this observation as well as the aforementioned requirements, the MVP for our part of the sprint goal was as follows:
```

\vspace{\baselineskip}

\textit{Make the minimally required amount of API endpoints in order to fulfill the needs for a working search functionality, and create the necessary documenation.}

\subsubsection{Backlog \& Increments}

> The next step of the planning phase is to granulate the sprint goal in to smaller task and then if possible further granulate these into subtasks. 

Reviewer: ```suggestion
The next step of the planning phase was to granulate the sprint goal into smaller tasks, and further granulate these into subtasks if possible.
```

A table of the tasks gained from granulating our sprint goal can be seen in table \ref{BacklogGranulationSprint3}. 
After granulating the tasks a session of planning poker was held. Each individual task was evaluated by the group with the time estimation added to the table. 
\begin{table}[h]
\centering
\begin{tabular}{|l|l|l|ll}
\cline{1-3}
WordCount API Granulation   & Subtasks & Estimation &  &  \\ \cline{1-3}
\multirow{4}{*}{Controller} & Get      & 2          &  &  \\ \cline{2-3}
                            & Post     & 5          &  &  \\ \cline{2-3}
                            & Update   & 8          &  &  \\ \cline{2-3}
                            & Delete   & 3          &  &  \\ \cline{1-3}
Docker                      &          & 13         &  &  \\ \cline{1-3}
Repository Service          &          & 10         &  &  \\ \cline{1-3}
CI/CD                       &          & 5          &  &  \\ \cline{1-3}
\end{tabular}
\caption{Backlog Granulation \& Estimation}
\label{BacklogEstimationSprint3}
\end{table}

> With the planning phase of the sprint completed the development phase can now begin. 

Reviewer: ```suggestion
With the planning phase of the sprint completed, the development phase can now begin. 
```
